fe/fe-core/src/main/java/com/starrocks/service/arrow/flight/sql/auth2/ArrowFlightSqlBasicCredentialValidator.java

```diff
@@ -27,40 +27,38 @@
 
 import java.nio.charset.StandardCharsets;
 
-public class ArrowFlightSqlCredentialValidator implements BasicCallHeaderAuthenticator.CredentialValidator {
+public class ArrowFlightSqlBasicCredentialValidator implements BasicCallHeaderAuthenticator.CredentialValidator {
 
-    private final ArrowFlightSqlTokenManager arrowFlightSqlTokenManager;
+    private final ArrowFlightSqlSessionManager sessionManager;
 
-    public ArrowFlightSqlCredentialValidator(ArrowFlightSqlTokenManager arrowFlightSqlTokenManager) {
-        this.arrowFlightSqlTokenManager = arrowFlightSqlTokenManager;
+    public ArrowFlightSqlBasicCredentialValidator(ArrowFlightSqlSessionManager sessionManager) {
+        this.sessionManager = sessionManager;
     }
 
     @Override
     public CallHeaderAuthenticator.AuthResult validate(String username, String password) throws Exception {
-        UserIdentity currentUser;
+        UserIdentity user;
         try {
-            currentUser = AuthenticationHandler.authenticate(new ConnectContext(), username, "0.0.0.0",
-                    password.getBytes(StandardCharsets.UTF_8));
+            user = AuthenticationHandler.authenticate(
+                    new ConnectContext(), username, "0.0.0.0", password.getBytes(StandardCharsets.UTF_8));
```

Comment:
new ConnectContext() is not appropriate, refer to the comment in createConnectContext